theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -34,6 +34,51 @@ def find_changed_packages(diff_range) {
     return changed_packages.split()
 }
 
+def find_changed_debs(diff_range) {
+    def dependencies = []
+    def plugins = []
+    def cores = []
```

Comment:
what's the idea behind noting them down individually, and then returning all together? ain't it "easier" (to the eyes) to `def changed_debs = []`, then in each `for …` do `changed_debs.add(…)` and later just `return changed_debs`?